Gamecube/menu/MenuContext.cpp

```diff
@@ -160,3 +163,13 @@ void MenuContext::draw()
 {
 	menu::Gui::getInstance().draw();
 }
+
+void MenuContext::Autoboot()
+{
+	if(strcasestr(AutobootPath,"sd:/") != NULL)
```

Comment:
You can pass down the argv[1] / argv[2] variables as arguments to the Autoboot() function, and Func_LoadFromSD / Func_LoadFromUSB(), then you don't need to use static buffers.